pkgs/applications/science/medicine/aliza/default.nix

```
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, wrapQtAppsHook
+, itk
+, cmake
+, qtbase
+}:
+
+stdenv.mkDerivation rec {
```

Please consider `finalAttrs`:

```suggestion
stdenv.mkDerivation (finalAttrs: {
```